src/parser.rs

```diff
@@ -4553,7 +4553,11 @@ impl<'a> Parser<'a> {
                 charset_name,
                 collation_name,
             })
-        } else if self.consume_token(&Token::Eq) || self.parse_keyword(Keyword::TO) {
+        } else if self.consume_token(&Token::Eq)
```

Comment:
Having tests would make it easier, but, tbh, I guess the SET TIME ZONE could be an individual command, no?

It is a singular command in many dialects. Having this manual conversion seems weird to me, considering that we can't have `SET TIME ZONE = ...` (I'm basing myself on [PostgreSQLs Docs](https://www.postgresql.org/docs/current/sql-set.html))